packages/button-theme/src/Card.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { applyTheme, StyleConfig } from '@button-inc/component-library/Card';
+import styles from './styles';
+
+const config: StyleConfig = {
+  defaultProps: {
+    size: 'medium',
```

Comment:
Should we add a storybook selector for changing the card size?

Reply:
I'm going to remove `size` as a prop instead. It's not part of the base component and bcgov doesn't use it. It's going to make the link font size look a little wonky, but I think that's fine for now since we're going to have to adjust all the styling once we have designs anyway

Reply:
Makes sense to me